File: SPA/src/app/_core/models/maintains/cham-cong.ts
export interface ChamCong {
  id: number;
  iD_NLD: number;
  iD_CD: number;
  quantity: number;
  date: string | Date;
}

export interface ChamCongDTO extends ChamCong {
  nld: string;
  cD_Name: string;
  idMaHang: number;
  mH_Name: string;
}

File: API/Models/ChamCong.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
     public partial class ChamCong
    {
        [Key]
        public int ID { get; set; }
        public int ID_NLD { get; set; }
        public int ID_CD { get; set; }
        public int Quantity { get; set; }
        public DateTime Date { get; set; }
       
    }
}